ApplyToBecomeInternal/ApplyToBecome.Data/Services/AcademyConversionProjectRepository.cs

```diff
@@ -23,11 +24,20 @@ public AcademyConversionProjectRepository(IHttpClientFactory httpClientFactory)
 			_httpClient = httpClientFactory.CreateClient("TramsClient");
 		}
 
-		public async Task<ApiResponse<ApiV2Wrapper<IEnumerable<AcademyConversionProject>>>> GetAllProjects(int page, int count, string statusFilters = "",
-			string titleFilter = "")
+		public async Task<ApiResponse<ApiV2Wrapper<IEnumerable<AcademyConversionProject>>>> GetAllProjects(int page, int count,
```

Comment:
It would be nice if we had a couple of unit tests covering this. But appreciate this is also covered by our intergration tests.